### DISCUSSION TITLE
Add /invsee toggle to /options

### DISCUSSION THREAD

**Discussion by PovilasLT** [Zrips/CMI#7044] [Enhancement]
Very useful function would be to toggle /invsee prevention. If true, other players should not be able to see inventory.

**Comment by MrCrazys** [comment]
You can already do this in the config.yml file of CMI, search for `Inv:` and find `BlackList:`. It would be unnecessary to add a toggle for /cmi inv, as you can already do it in the config.yml file. Besides, operators bypass this option anyways.
